modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -426,4 +426,15 @@ public static class GarbageCollector {
         /** Garbage collector closed error. */
         public static final int CLOSED_ERR = GC_ERR_GROUP.registerErrorCode(1);
     }
+
+    /**
+     * Authentication error group.
+     */
+    public static class Authentication {
+        /** Authentication error group. */
+        public static final ErrorGroup AUTHENTICATION_ERR_GROUP = ErrorGroup.newGroup("AUTHENTICATION", 15);
+
+        /** General authentication error. */
+        public static final int UNKNOWN_AUTHENTICATION_ERR = AUTHENTICATION_ERR_GROUP.registerErrorCode(1);
```

Comment:
In case this is the general auth error, why the variable is called `UNKNOWN_*`? Looking at the rest of the code we can see such variables called `COMMON_*`.